### DISCUSSION TITLE
GitHub Classroom New Assignment Broken

### DISCUSSION THREAD

**Discussion by williamstudentqm** [community/community#28083] [GitHub Education]
Whenever I click on the hyperlink generated by accepting a new assignment in GitHub classroom I get a 404 error. 
![image](https://user-images.githubusercontent.com/110464268/182420286-ed1a73d1-c8d1-4dae-82e6-c9e9d5110676.png)

**Comment by juanpflores** [comment]
cc @divais 👀

**Comment by JessRudder** [comment]
Hi @williamstudentqm is this happening as soon as you accept the assignment? The screenshot above appears to be a repo created for a different user than your own (joeystudentqm). Unless you were that user or an admin on that org, you would not have access to it (in which case the 404 response is the correct response).

If you log in as joeystudentqm (assuming that's a user you have access to) or you are logged in as an admin on the org the link should work. If that is not the case, please let us know.
